lib/simulator-xcode-9.js

```diff
@@ -64,6 +66,11 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     if (opts.scaleFactor) {
       opts.devicePreferences.SimulatorWindowLastScale = parseFloat(opts.scaleFactor);
     }
+    if (opts.simWindowCenter && opts.simWindowCenter.x && opts.simWindowCenter.y) {
+      opts.devicePreferences.SimulatorWindowCenter = `{${opts.simWindowCenter.x},${opts.simWindowCenter.y}}`;
+    } else {
+      log.warn(`Invalid simWindowCenter value: ${opts.simWindowCenter}. Valid format {x:x, y:y}`);
```

Comment:
this warning will be always shown even if _simWindowCenter_ is not set. I don't think this is the expected behaviour.